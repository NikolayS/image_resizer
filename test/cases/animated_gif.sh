#!/bin/bash

# to run this scenario, RESIZERSERVICE should be defined (`export RESIZERSERVICE=https://your.resizer.host.name`)

http GET "$RESIZERSERVICE/?w=200&src=https://upload.wikimedia.org/wikipedia/commons/2/2c/Rotating_earth_(large).gif" -h | grep -v "Date:"
