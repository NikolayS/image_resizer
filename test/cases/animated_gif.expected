HTTP/1.1 200 OK
Server: nginx
Content-Type: image/gif
Transfer-Encoding: chunked
Connection: keep-alive
X-Content-Length-Original: 1429302
Content-Lenght: 590368

